registry/auth/token/fuzz_test.go

```diff
@@ -6,29 +6,29 @@ import (
 	fuzz "github.com/AdaLogics/go-fuzz-headers"
 )
 
-func FuzzToken1(f *testing.F) {
-	f.Fuzz(func(t *testing.T, data []byte) {
-		ff := fuzz.NewConsumer(data)
-		rawToken, err := ff.GetString()
-		if err != nil {
-			return
-		}
-		verifyOps := VerifyOptions{}
-		err = ff.GenerateStruct(&verifyOps)
-		if err != nil {
-			return
-		}
-		token, err := NewToken(rawToken)
-		if err != nil {
-			return
-		}
-		_, err = token.Verify(verifyOps)
-		if err != nil {
-			return
-		}
-		_, _ = token.VerifySigningKey(verifyOps)
-	})
-}
+//func FuzzToken1(f *testing.F) {
```

Comment:
This has been commented out I'm not exactly sure what signing algorithm I should pass into `NewToken`. Maybe @AdamKorcz could help here

Reply:
Does it matter? Can't we just pick a common one?

Reply:
> Does it matter? Can't we just pick a common one?

That would be fine. Or have all in a slice and have the fuzzer decide which one. The `f.Fuzz` call could be `f.Fuzz(func(t *testing.T, data []byte, algEnum uint) {`, and the fuzzer could select the alg from the slice like this: `alg := algSlice[algEnum%len(algSlice)]`